### PR TITLE
Validators

### PR DESCRIPTION
**Pull request by ztsai**
- check if all validators are in db, insert one if not
- store keybase profile url in db

### CODE REVIEW

**Comment by ztsai** [comment]
#93

File: imports/api/blocks/server/methods.js [hunk 1]
```diff
@@ -482,6 +481,79 @@ Meteor.methods({
...
+                                        bulkValidators.find({consensus_pubkey: conPubKey}).upsert().updateOne({$set:validator_data});
+                                    }
+                                })
+                            } catch (e){
+                                console.log(e)
+                            }
+                        }
+
+                        // fetching keybase every 14400 blocks(~1 day)
+                        if (height % 14400 == 1){
```
Comment (kwunyeung) [inline imports/api/blocks/server/methods.js]: Possible to let validators to show their profile picture after they have updated their validator info? Validators expect to see it immediately. Store the profile url when a `create-validator` or `edit-validator` tx happen?

Comment (ztsai) [reply imports/api/blocks/server/methods.js]: ack

**Comment by ztsai** [comment]
personally I think it's fine, currently we are not able to show their profile picture most of the time due to the 429 error to keybase.io anyway
but I can write a migration if you think it's necessary

**Pull request merged**